lib/checks/forms/autocomplete-appropriate-evaluate.js

```diff
@@ -12,6 +12,7 @@ function autocompleteAppropriateEvaluate(node, options, virtualNode) {
   const allowedTypesMap = {
     bday: ['text', 'search', 'date'],
     email: ['text', 'search', 'email'],
+    username: ['email'],
```

Comment:
I believe to fix the test we'll need this to also include `text` and probably `search`. Otherwise it's restricted to just `type=email`.

Reply:
Ah! That makes sense, thanks for the hint. :)